enterprise/api_client/enterprise_catalog.py

```diff
@@ -200,14 +200,35 @@ def delete_enterprise_catalog(self, catalog_uuid):
             )
             return {}
 
+    def traverse_get_content_metadata(self, endpoint, query, catalog_uuid):
+        """
+        Helper method to traverse over a paginated response from the enterprise-catalog service's `get_content_metadata`
+        endpoint.
+        """
+        content_metadata = OrderedDict()
+        try:
+            response = endpoint.get(**query)
+            for item in utils.traverse_pagination(response, endpoint):
+                content_id = utils.get_content_metadata_item_id(item)
+                content_metadata[content_id] = item
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
+                'Failed to get content metadata for Catalog [%s] in enterprise-catalog due to: [%s]',
+                catalog_uuid, str(exc)
```

Comment:
nit: I believe `LOGGER.exception` already logs the exception without needing to explicitly include it in the message?

Reply:
```
There are three keyword arguments in kwargs which are inspected: exc_info which, if it does not evaluate as false, causes exception information to be added to the logging message.
```

Reply:
yup I believe exc_info is the desired method?